azurerm/helpers/azure/zones.go

```diff
@@ -1,14 +1,18 @@
 package azure
 
-import "github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azuread/azuread/helpers/validate"
```

Comment:
Looks like this got imported from the wrong provider :)

Reply:
Ohhh, good catch

Reply:
Fixed!